server/executor/runner.go

```diff
@@ -192,6 +221,11 @@ func (r persistentRunner) processExecQueue(job execReq) {
 		}
 	}
 
+	err = r.eventEmitter.Emit(job.ctx, events.TriggerCreatedInfo(job.run.TestID, job.run.ID))
```

Comment:
It makes sense to trigger the `TriggerExecutionStart`, right?

```suggestion
	err = r.eventEmitter.Emit(job.ctx, events.TriggerExecutionStart(job.run.TestID, job.run.ID))
```